bin/spacewalk-uln-resolver.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2019, SUSE Linux GmbH.
```

Comment:
This is new code and we should follow the current copyright assignment guidelines.

>Copyright
>
>The code you contribute upstream or publish as Open Source as part of your job at SUSE in your work time is copyrighted or otherwise owned by SUSE. Use “SUSE LLC” when attributing the copyright. Use your “@suse.com” email address for contributions.
>
>[SUSE Open Source Policy](https://opensource.suse.com/suse-open-source-policy) (Copyright section):